packages/core-datepicker/core-datepicker.jsx

```diff
@@ -34,8 +34,8 @@ export default class Datepicker extends React.Component {
 
 // Expose API and config
 Datepicker.parse = coreDatepicker.parse
-Datepicker.months = coreDatepicker.months
-Datepicker.days = coreDatepicker.days
+Datepicker.setMonths = (months) => (coreDatepicker.months = months)
```

Comment:
if this is a function in JSX, should it be so in vanilla as well for consistency?

Reply:
Hehe, @mia-aasbakken and me were discussing that and thought we would see if you would comment on it :p. We didn't want to say anything as to not influence your opinion, but since you also favor consistency I'll do the change :) !